### ISSUE TITLE
TLS support

### ISSUE THREAD

**Issue by devtransition** [easternbloc/node-stomp-client#26]
The STOMP server we are connecting just supports TLS/SSL connections.
...but the client seems not to support that.

Is that feature planed or is there a fork that already supports encrypted connections?

**Comment by dvtylr** [comment]
The SecureStompClient code no longer worked as it relied on an API that is no longer available in node.  I have created a PR for TLS support.  Necessary changes were minor but I also allowed for the constructor to accept options as an object so I did not need to pass in so many nulls.

**Comment by mantoni** [comment]
I'm connection with SSL on node 0.10.38. Works fine for me:

```js
new stomp.Stomp({
  ssl: true,
  ssl_options: {
    ca: [sslCert]
  },
  // the other connection options
});
```

**Comment by dvtylr** [comment]
Are you sure you are using this module?  The syntax you are using looks like it is for stomp-js and not stomp-client.

**Comment by mantoni** [comment]
Oh, I'm very sorry. You're right @dvtylr.

**Comment by easternbloc** [comment]
Feature available in 0.7.0 added by @dvtylr closing this ticket now.
